index.html

```diff
@@ -122,13 +124,14 @@
       </div>
     </div>
   </div>
-  <div id="main" class="container-fluid">
-  </div>
+  <div id="main" class="container-fluid"></div>
   <div id="modal" class="modal fade">
     <div class="modal-dialog">
       <div class="modal-content">
       </div>
     </div>
   </div>
+  <div class="panel-manager">
```

Comment:
Umm looks like panel-manager is a controller class in code and this could be "side-panel" instead...